specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensions.json

```diff
@@ -0,0 +1,659 @@
+            },
+            "version": {
+              "description": "Version of the extension for this extension, if it is 'pinned' to a specific version. autoUpgradeMinorVersion must be 'false'.",
+              "type": "string"
+            },
+            "scope": {
+              "description": "Scope at which the extension is installed.",
+              "$ref": "#/definitions/Scope"
+            },
+            "configurationSettings": {
+              "description": "Configuration settings, as name-value pairs for configuring this extension.",
+              "type": "object",
+              "x-nullable": true,
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "configurationProtectedSettings": {
+              "description": "Configuration settings that are sensitive, as name-value pairs for configuring this extension.",
+              "type": "object",
+              "x-nullable": true,
+              "x-ms-secret": true,
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "provisioningState": {
+              "description": "Status of installation of this extension.",
+              "type": "string",
+              "readOnly": true,
+              "$ref": "#/definitions/provisioningState"
+            },
+            "statuses": {
+              "description": "Status from this extension.",
+              "type": "array",
+              "x-nullable": true,
+              "items": {
+                "$ref": "#/definitions/ExtensionStatus"
+              }
+            },
+            "errorInfo": {
+              "description": "Error information from the Agent - e.g. errors during installation.",
+              "type": "object",
+              "readOnly": true,
+              "x-nullable": true,
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorDetail"
+            },
+            "customLocationSettings": {
+              "description": "Custom Location settings properties.",
+              "type": "object",
+              "readOnly": true,
+              "x-nullable": true,
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "packageUri": {
+              "description": "Uri of the Helm package",
+              "type": "string",
+              "readOnly": true,
+              "x-nullable": true
+            }
+          }
+        },
+        "identity": {
+          "description": "Identity of the Extension resource",
+          "x-nullable": true,
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Identity"
+        },
+        "systemData": {
+          "description": "Top level metadata https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources",
```

Comment:
readonly

Reply:
Similar to the previous comment, in CommonTypes.json, where systemData is defined, it is already marked as readOnly.